feature/map/src/main/java/io/github/droidkaigi/confsched2022/feature/map/Map.kt

```diff
@@ -55,10 +63,27 @@ fun Map(
                 .padding(innerPadding),
             contentAlignment = Alignment.Center
         ) {
+            val minScale = 1f
+            val maxScale = 3f
+            var scale by remember { mutableStateOf(minScale) }
+            var offset by remember { mutableStateOf(Offset.Zero) }
+            val state = rememberTransformableState { zoomChange, offsetChange, _ ->
+                scale *= zoomChange
+                offset += offsetChange
+            }
             Image(
                 painter = painterResource(id = R.drawable.map),
                 contentDescription = "Floor map",
-                modifier = Modifier.padding(16.dp)
+                modifier = Modifier
+                    .transformable(state = state)
+                    .graphicsLayer(
+                        scaleX = maxOf(minScale, minOf(maxScale, scale)),
```

Comment:
How about:

```suggestion
                        scaleX = scale.coerceIn(minScale, maxScale),
```

Reply:
Thank you for your review!
I will take in it.